spec/acceptance/email_notification_spec.rb

```diff
@@ -52,6 +54,14 @@ def post_notification
         post_notification
         expect(last_response.body).to eq('')
       end
+
+      describe 'POSTing a Amazon SES Setup Notification to /user-signup/email-notification' do
```

Comment:
I think the wrapping block already gives most of the context, so this could be terser?

```suggestion
      describe 'of the "Amazon SES Setup" type' do
```